src/LegacyRepContract.sol

```diff
@@ -0,0 +1,82 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/legacy_reputation/StandardToken.sol';
+import 'ROOT/legacy_reputation/Ownable.sol';
+import 'ROOT/Controller.sol';
+
+
+contract LegacyRepContract is StandardToken, Controlled, Ownable {
```

Comment:
This should be a `VariableSupplyToken` as we'll be increasing total supply anytime someone wants REP.

Reply:
Ah ok. We don't need to allocate a balance for the token itself then if we're doing that. Will do!

Reply:
Also, I don't think this contract needs to be Controlled or Ownable.  Just `VariableSupplyToken`.